.github/workflows/python-package.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: []
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
```

Comment:
Can we use pylint? theres already .pylintrc

Reply:
Also why add another workflow? you could have just updated the existing one run run flake8/black since there aren't any other differences

Reply:
Yeah probably should be just one workflow

Reply:
I see now, I wasn't really paying attention to the new workflow file I just used github actions and only changed the 'on' and python version values.

Reply:
It seems that the workflow doesn't really do anything so I'll just remove it. I thought it would run `pip install .` or something.